src/lib/output/themes/default/partials/footer.tsx

```diff
@@ -1,18 +1,23 @@
-import { JSX } from "../../../../utils";
+import { JSX, Raw } from "../../../../utils";
 import type { DefaultThemeRenderContext } from "../DefaultThemeRenderContext";
 
 export function footer(context: DefaultThemeRenderContext) {
     const hideGenerator = context.options.getValue("hideGenerator");
+    const customFooter = context.options.getValue("footer");
     return (
         <footer>
             {context.hook("footer.begin")}
-            {hideGenerator || (
-                <p class="tsd-generator">
-                    {"Generated using "}
-                    <a href="https://typedoc.org/" target="_blank">
-                        TypeDoc
-                    </a>
-                </p>
+            {customFooter ? (
```

Comment:
Hmm... it's unfortunate that there are now two options to control the generated footer... Something I probably want to change during the next round of breaking changes.